src/test/java/com/tngtech/java/junit/dataprovider/internal/convert/ObjectArrayConverterTest.java

```diff
@@ -38,7 +38,7 @@ public void testConvertShouldCreateEmptyVarargsArrayForMissingOnlyVarargsArgumen
         Object[] result = underTest.convert(data, true, parameterTypes);
 
         // Then:
-        assertThat(result).isEqualTo(new Object[] { new int[0] });
+        assertThat(result).containsExactly(new int[0]);
```

Comment:
Here I really prefer the ```isEqualTo``` to check for the ```Object[]``` which is wrapping the real result. This is in my humble opinion clearer. Though, this is just a gut feeling ...